enterprise/frontend/src/embedding/data-picker/DataSelector.jsx

```diff
@@ -672,22 +571,13 @@ export class UnconnectedDataSelector extends Component {
     const loadersForSteps = {
       // NOTE: make sure to return the action's resulting promise
       [DATA_BUCKET_STEP]: () => {
-        return Promise.all([
```

Comment:
and since we now only have 1 promise we don't need to have `Promise.all` anymore.